app.go

```diff
@@ -195,6 +196,14 @@ func (a *App) Setup() {
 		a.ErrWriter = os.Stderr
 	}
 
+	// add global flags added by other packages
+	flag.VisitAll(func(f *flag.Flag) {
+		// skip test flags
+		if !strings.HasPrefix(f.Name, "test.") {
+			a.Flags = append(a.Flags, extFlag{f})
```

Comment:
I'm good with specifically ignoring flags with `"test."` prefix in this implementation 👍🏼 I looked briefly and couldn't find any exported code that could be used to get this prefix from go libraries 🤷🏼 

What's the reasoning behind using a `extFlag{f}` instead of `&extFlag{f}` and having the `extFlag` methods use a pointer receiver?

Reply:
We need the test prefix check because otherwise when we run unit tests those flags get added !!!!

Reply:
@dearchap I'm in favor of filtering out `"test." in general 👍🏼  My thinking was more around if we could read this value from somewhere instead of having it hard-coded here 🤷🏼 

As for `extFlag`, I meant that I wondered if you meant to use the struct and method receivers as structs rather than pointers to structs. In this implementation, there's going to be a lot more implicit copying of structs, IIUC.

Reply:
@meatballhat do you want it to be defined as an App field or as a library var ?

Reply:
> @meatballhat do you want it to be defined as an App field or as a library var ?

@dearchap Maybe he means use pointer receivers(*extFlag) rather than value receivers(extFlag) to avoid unnecessary data duplication.

Reply:
@meatballhat I updated the extFlag to be pointer receiver.